src/pydvl/value/shapley/naive.py

```diff
@@ -1,17 +1,12 @@
 import math
 import warnings
 from itertools import permutations
+from typing import List
 
 import numpy as np
+from numpy.typing import NDArray
```

Comment:
Why don't we always directly import like this instead of using the if TYPING guard and then quoting all types? (that's a bit of a PITA, tbh)

Reply:
I think it was a leftover from when we didn't pin the minimum numpy version to 1.20.

The [typing](https://numpy.org/devdocs/reference/typing.html) module was added to numpy in version 1.20.

So, yes I think we should just import it directly.